tests/converters/herrenberg_bike_test.py

```diff
@@ -0,0 +1,86 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from pathlib import Path
+from unittest.mock import Mock
+
+import pytest
+from parkapi_sources.converters import HerrenbergBikePullConverter
+from requests_mock import Mocker
+
+from tests.converters.helper import validate_realtime_parking_site_inputs, validate_static_parking_site_inputs
+
+
+@pytest.fixture
+def requests_mock_herrenberg_bike(requests_mock: Mocker) -> Mocker:
+    json_path = Path(Path(__file__).parent, 'data', 'herrenberg_bike.json')
+    with json_path.open() as json_file:
+        json_data = json_file.read()
+
+    requests_mock.get('https://www.munigrid.de/api/file/11/latest', text=json_data)
+
+    return requests_mock
+
+
+@pytest.fixture
+def herrenberg_bike_config_helper(mocked_config_helper: Mock):
+    config = {}
+    mocked_config_helper.get.side_effect = lambda key, default=None: config.get(key, default)
+    return mocked_config_helper
+
+
+@pytest.fixture
+def herrenberg_bike_pull_converter(herrenberg_bike_config_helper: Mock) -> HerrenbergBikePullConverter:
+    return HerrenbergBikePullConverter(config_helper=herrenberg_bike_config_helper)
+
+
+@pytest.fixture
+def herrenberg_bike_ignore_missing_capacity_config_helper(mocked_config_helper: Mock):
+    config = {
+        'PARK_API_HERRENBERG_BIKE_IGNORE_MISSING_CAPACITIES': True,
+    }
+    mocked_config_helper.get.side_effect = lambda key, default=None: config.get(key, default)
+    return mocked_config_helper
+
+
+@pytest.fixture
+def herrenberg_bike_ignore_missing_capacity_pull_converter(
+    herrenberg_bike_ignore_missing_capacity_config_helper: Mock,
+) -> HerrenbergBikePullConverter:
+    return HerrenbergBikePullConverter(config_helper=herrenberg_bike_ignore_missing_capacity_config_helper)
+
+
+class HerrenbergBikePullConverterTest:
+    @staticmethod
+    def test_get_static_parking_sites(herrenberg_bike_pull_converter: HerrenbergBikePullConverter, requests_mock_herrenberg_bike: Mocker):
+        static_parking_site_inputs, import_parking_site_exceptions = herrenberg_bike_pull_converter.get_static_parking_sites()
+
+        assert len(static_parking_site_inputs) == 184
+        assert len(import_parking_site_exceptions) == 0
```

Comment:
Something is wrong here: without `PARK_API_HERRENBERG_BIKE_IGNORE_MISSING_CAPACITIES` being set to true, this should not give 0 exceptions.

Reply:
Since there are no exceptions from the data import, setting ```PARK_API_HERRENBERG_BIKE_IGNORE_MISSING_CAPACITIES``` to ```True``` or ```False``` results in 0 exceptions. or did I get you wrong?

Reply:
Hmmm. Maybe the question is: why are there 170 successfully converted datasets at https://github.com/ParkenDD/parkapi-sources-v3/pull/110/files/93939ac2c9f71d3b4bad8a172b40430b92699c01#diff-afbaa11190a7f0deebe5561430404cb2c4ed2bcea277cebe41b4059a6f4d7e63 , but 184 now? Did Herrenberg fix the missing capacities?

Reply:
Oh now I get. After sending the Excel-Reference table for Bike parking to Herrenberg, the developer at Munigrid fixed the data and its attributes. This is the bike parking data [before](https://github.com/ParkenDD/parkapi-sources-v3/pull/110/files/93939ac2c9f71d3b4bad8a172b40430b92699c01#diff-aa8cb365059fd974bb4f74aa3aac42339cbbd3941cc30ec6f7c137a76924c47c) and the fixed data now look like this [now](https://github.com/ParkenDD/parkapi-sources-v3/pull/110/files#diff-aa8cb365059fd974bb4f74aa3aac42339cbbd3941cc30ec6f7c137a76924c47c).

Reply:
Ok, then the ignore feature flag is basically untestable atm, because there are no broken datasets any more? Fine for me.